user-guide-bianca.md

```diff
@@ -0,0 +1,71 @@
+# Encryption of files for EGA-SE in Bianca
+This document is a how-to-guide on encrypting files for the EGA-SE service . It is meant for submitters to EGA-SE service and it assumes the files are stored in Bianca. EGA provides permanent secure archiving and sharing of all types of potentially identifiable human genetic and phenotypic data. Submitted files need to be encrypted according to the [crypt4gh](https://www.ga4gh.org/news/crypt4gh-a-secure-method-for-sharing-human-genetic-data/) standard.
+
+## Pull the images with singularity
+
+### Pull images
+* Login to Rackham running:
+```bash
+ssh <username>@rackham.uppmax.uu.se
+```
+Navigate to the folder you want to pull the images.
+* To download the crypt4gh key generator image run the following command in the terminal:
+```bash
+singularity pull docker://nbisweden/ega-crypt4gh:crypt4gh-keygen-1.0.0
+```
+* To download the crypt4gh image run the following command in the terminal:
+```bash
+singularity pull docker://nbisweden/ega-crypt4gh:crypt4gh-1.0.0
+```
+Now you should be able to see two files with `.sif` extensions in the folder when running the `ls` command.
+
+### Move the singularity images to Bianca
+You can move the singularity images to Bianca using wharf. The guide for transferring files from and to Bianca can be found [here](https://www.uppmax.uu.se/support/user-guides/transit-user-guide/).
+
+In summary, the steps needed are the following:
+In a terminal window
+* Login to transit running:
+```bash
+ssh <username>@transit.uppmax.uu.se
+```
+* Mount to the bianca project folder using the following command
+```bash 
+mount_wharf <project_name>
+```
+where [project_name] in the form of sens20XXX.
+* Copy image files using the cp command and replacing to the correct locations:
+```bash 
+cp ega-crypt4gh_crypt4gh-1.0.0.sif <project_name>/
+cp ega-crypt4gh_crypt4gh-keygen-1.0.0.sif <project_name>/
+```
+
```

Comment:
New heading here?

And for the transfer, need to bring in the ega-se key?

Reply:
Yes, that's on one of the next sections